packages/backend/src/queryBuilder.ts

```diff
@@ -266,7 +263,7 @@ export const getCustomDimensionSql = ({
                         sortField.fieldId,
                 );
             const quoteChar = warehouseClient.getStringQuoteChar();
-            const dash = `${quoteChar}-${quoteChar}`;
+            const dash = `${quoteChar} - ${quoteChar}`;
```

Comment:
adding space before and after `-` coz it looked weird for negative ranges

Reply:
That makes sense, you'll have to update some e2e tests after this